plugins/linuxv2/model/namespace/namespace.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package namespace;
+
+message Namespace {              /* Linux network namespace */
```

Comment:
This is debatable.. All other linux models now have `Linux` prefix and this one does not, also it should probably be more clear that it's network namespace since there are other types of namespaces in linux. Going with `LinuxNetworkNamespace` might be too long, but we could shorten it to: `LinuxNetNamespace`, this will also remove the stutter: `namespace.Namespace..`.

Reply:
Isn't that what I suggested?

Reply:
yep, sorry, didn't read it right :)